src/imperator/data/provinces.rs

```diff
@@ -64,6 +64,7 @@ impl ImperatorProvinces {
                             || key.is("river_provinces")
                             || key.is("impassable_terrain")
                             || key.is("uninhabitable")
+                            || key.is("wasteland")
                             || key.is("lakes")
                             || key.is("LAKES")
                         {
```

Comment:
Does this mean they should all really be key.lowercase_is ? To do a case insensitive comparison

Reply:
Yeah that would probably work better good call